app/controllers/meetings_controller.rb

```diff
@@ -91,18 +120,18 @@ def destroy
     end
   end
 
-  # def check_employee_overlap!
-  #   @attendees = EmployeeMeeting.where(meeting_id: params[:id])
-  #   @attendees.each do |attendee|
-  #     @begin_time = attendee.meeting.start_time
-  #     @finish_time = attendee.meeting.end_time
-  #   end
-  #   if @meeting.start_time< Time.now
-  #     redirect_to @meeting, alert: 'You are already in another meeting.'
-  #   end
-  # end
-
-
+  # An employee cannot create or join a meeting that takes place during another meeting they own or are part of.
+  def meeting_overlap? (check_meeting)
+    all_meetings = Meeting.where(employee_id: current_employee.id) # owned meetings
+    all_meetings += EmployeeMeeting.where(employee_id: current_employee.id).map{|em| em.meeting} # attending meetings
+    all_meetings.each do |meeting|
+      if (meeting.start_time <= check_meeting.start_time and check_meeting.start_time <= meeting.end_time) or
```

Comment:
We don't want to ever use `and` in ruby and instead use `&&`. http://devblog.avdi.org/2010/08/02/using-and-and-or-in-ruby/